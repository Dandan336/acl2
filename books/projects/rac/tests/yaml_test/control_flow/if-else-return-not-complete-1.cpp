// RAC begin

int foo(int a)
{
  if (a)
    return 1;
  else
    a = 2;

  return 0;
}

// RAC end
