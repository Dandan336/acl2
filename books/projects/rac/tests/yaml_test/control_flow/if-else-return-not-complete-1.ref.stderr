if-else-return-not-complete-1.cpp:8: syntax error, unexpected ID, expecting IF or RETURN or '{'
